examples/extensions-zalando/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Apr 04 15:18:10 CEST 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
```

Comment:
Any reason why you switched from `all` to `bin`?

Reply:
That's actually the default created by the wrapper, i.e. just the runtime. I guess the use cases for bundled Gradle sources and docs are very limited (and sources/docs can always be downloaded individually if needed).

Reply:
Yes, I know, that you have to call the wrapper with a specific parameter to get the `all` distribution. I just remember that when you open a project in IntelliJ with just the `bin` wrapper version, it recommends to use the `all` distribution. But to be fair, I don't know why.

Reply:
I think it was something about better developer experience.

Reply:
Based on some old discussions it should help when editing build.gradle. However, with current IDEA version at least with build.gradle.kts it seems to work with the -bin distribution too:
<img width="360" alt="Screen Shot 2023-03-15 at 3 29 50 PM" src="https://user-images.githubusercontent.com/8832826/225325819-ffdf090b-8bdf-4cd1-a5d4-1774ff22f428.png">
Also IDEA docs didn't seem to have any mention of that (https://www.jetbrains.com/help/idea/gradle.html). So, perhaps it's not relevant nowadays. Anyway, happy to change if preferred.